app/assets/stylesheets/user_style.css

```diff
@@ -82,6 +82,10 @@ a {
   padding: 10px 10px;
 }
 
+/* .user-name > a {
+  color: #fff;
+} */
+
```

Comment:
## OPTIONAL
- [ ] kindly remove commented code.

Reply:
Thanks so much Mr @andersonlebon  for professional review